src/map/npc.c

```diff
@@ -3751,11 +3751,16 @@ static const char* npc_parse_mob(char* w1, char* w2, char* w3, char* w4, const c
 	mob.ys = (signed short)ys;
 	if (mob_lv > 0 && mob_lv <= MAX_LEVEL)
 		mob.level = mob_lv;
-	if (size > 0 && size <= 2)
+	if (size > SZ_SMALL && size <= SZ_BIG)
 		mob.state.size = size;
 	if (ai > AI_NONE && ai <= AI_MAX)
 		mob.state.ai = ai;
 
+	if (mob.xs < 1)
+		mob.xs = 0;
```

Comment:
```xs``` and ```ys``` short int not initialized https://github.com/rathena/rathena/blob/1bf5e69416c30fb13d4eadc27fced2609dd20589/src/map/npc.c#L3691
and ```mob.xs = (signed short)xs;```.

Any idea to distinguish none input (always negative) from negative input?